app.py

```diff
@@ -712,8 +730,7 @@ async def helper(
             embed = discord.Embed()
         embed.set_author(name=f"{str(interaction.user)}", icon_url=interaction.user.display_avatar.url)
         await interaction.send(helper_role.mention, embed=embed)
-        return
```

Comment:
Was this a mistake?

Reply:
Idk bro it was already there I did not add it

Reply:
You have remove the `return`.

Reply:
Oh maybe that was an accident. I don't remember doing that lol sorry.